app/uk/gov/hmrc/servicedependencies/controller/ServiceDependenciesController.scala

```diff
@@ -216,30 +220,33 @@ class ServiceDependenciesController @Inject()(
                                      RepositoryModule(
                                        name                = m.name,
                                        group               = m.group,
-                                       dependenciesCompile = compileDependencies ++
-                                                               dependencyIfMissing(
-                                                                 compileDependencies,
-                                                                 group    = "org.scala-lang",
-                                                                 artefact = "scala-library",
-                                                                 versions = scalaVersions,
-                                                                 scope    = DependencyScope.Compile
-                                                               ),
-                                       dependenciesTest    = testDependencies ++
-                                                               dependencyIfMissing(
-                                                                 testDependencies,
-                                                                 group    = "org.scala-lang",
-                                                                 artefact = "scala-library",
-                                                                 versions = scalaVersions,
-                                                                 scope    = DependencyScope.Test
-                                                               ),
-                                       dependenciesIt      = itDependencies ++
-                                                               dependencyIfMissing(
-                                                                 testDependencies,
-                                                                 group    = "org.scala-lang",
-                                                                 artefact = "scala-library",
-                                                                 versions = scalaVersions,
-                                                                 scope    = DependencyScope.It
-                                                               ),
+                                       dependenciesCompile = when(compileDependencies.nonEmpty) {
+                                                               compileDependencies ++ dependencyIfMissing(
```

Comment:
I think it's clearer with `compileDependencies ++ when(..)(dependencyIfMissing)`